pypeit/core/flexure.py

```diff
@@ -462,7 +467,7 @@ def spec_flex_shift_global(slit_specs, islit, sky_spectrum, arx_fwhm_pix, empty_
     flex_dict = copy.deepcopy(empty_flex_dict)
 
     # Calculate the shift
-    fdict = spec_flex_shift(slit_specs[islit], sky_spectrum, arx_fwhm_pix, mxshft=mxshft, excess_shft=excess_shft,
+    fdict = spec_flex_shift(slit_specs[islit], sky_file, mxshft=mxshft, excess_shft=excess_shft,
```

Comment:
as `sky_file=="model"` requires the `obj_sky_spec` this method can't use that option, right?   just checking